### PR TITLE
Add Edit Profile modal for individual user

### PR DESCRIPTION
**Pull request by subinasr**
depends on https://github.com/toggle-corp/kitab-server/pull/22

### CODE REVIEW

File: app/views/Profile/IndividualProfile/EditProfileModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+                        variant="primary"
+                        onClick={submit}
+                        disabled={pristine || updateProfilePending}
+                    >
+                        Save
+                    </Button>
+                </>
+            )}
+        >
+            <div className={styles.description}>
```
Comment (AdityaKhatri) [inline app/views/Profile/IndividualProfile/EditProfileModal/index.tsx]: unnecessary div

File: app/views/Profile/IndividualProfile/EditProfileModal/styles.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.edit-modal {
+    display: flex;
+}
```
Comment (AdityaKhatri) [inline app/views/Profile/IndividualProfile/EditProfileModal/styles.css]: unnecessary style page

File: app/views/Profile/IndividualProfile/index.tsx [hunk 3]
```diff
@@ -95,42 +99,62 @@ function OrderListRenderer(props: OrderListProps) {
     } = props;
 
     return (
-        <Link to={generatePath(routes.orderList.path, { activeOrderId: orderCode })}>
-            <Card
+        <Link
```
Comment (AdityaKhatri) [inline app/views/Profile/IndividualProfile/index.tsx]: Use SmartLink

File: app/views/Profile/IndividualProfile/index.tsx [hunk 4]
```diff
@@ -143,8 +167,8 @@ function IndividualProfile() {
     );
 
     const handleEditProfile = useCallback(() => {
-        console.warn('handle me');
-    }, []);
+        showEditProfileModal();
+    }, [showEditProfileModal]);
```
Comment (AdityaKhatri) [inline app/views/Profile/IndividualProfile/index.tsx]: use directly

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

File: app/views/Profile/IndividualProfile/index.tsx (hunk 4 above)
Comment (frozenhelium) [reply app/views/Profile/IndividualProfile/index.tsx]: Wouldn't it be better this way @AdityaKhatri ?
```jsx
<Button
  name={undefined}
  variant="general"
  onClick={showEditProfileModal}
  icons={<IoPencil />}
>
  Edit Profile
</Button>
```

Comment (AdityaKhatri) [reply app/views/Profile/IndividualProfile/index.tsx]: yes, my comment exactly :+1:

**Pull request merged**